### PR TITLE
Use Cloudformation signals

### PR DESCRIPTION
**Pull request by fiunchinho**
Can you take a look @aramirez-es @Hyunk3l @ismFerDev @danielricart?

### CODE REVIEW

File: files/healthcheck.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+function cfn-signal() {
+  /usr/bin/docker run --rm mbabineau/cfn-bootstrap cfn-signal \
+    -e $? \
+    --stack ${STACK_NAME} \
+    --resource ${RESOURCE_NAME} \
+    --region ${AWS_REGION}
+}
+trap cfn-signal EXIT
+
+sleep 5 # It takes compose a few seconds to create the container
```
Comment (Hyunk3l) [inline files/healthcheck.sh:19]: First time would download and run it...are 5 seconds enough?

**Review by ismferd — APPROVED** [review]

File: troposphere/asg-elb-dns.py [hunk 2]
```diff
@@ -32,7 +32,8 @@ def generate_cloudformation_template():
     ))
 
     dns_ttl = template.add_parameter(Parameter(
-        "TTL",
+        "DNSTTL",
```
Comment (Hyunk3l) [inline troposphere/asg-elb-dns.py:35]: Could you link documentation about this please?

File: files/healthcheck.sh (hunk 1 above)
Comment (fiunchinho) [reply files/healthcheck.sh:19]: I want to test it again, just to make sure, but I believe Compose does a weird order of execution when starting containers

File: troposphere/asg-elb-dns.py (hunk 2 above)
Comment (fiunchinho) [reply troposphere/asg-elb-dns.py:35]: Sure!

File: files/healthcheck.sh (hunk 1 above)
Comment (ismaelfernandezscmspain) [reply files/healthcheck.sh:19]: the function cfn-signal will run when it caught or 'trap' an EXIT signal.

File: files/healthcheck.sh [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+# If no healthcheck has been defined, use regular container status
+if [[ $(docker inspect ${STACK_NAME} | jq '.[].State.Health|length') == 0 ]]; then
```
Comment (aramirez-es) [inline files/healthcheck.sh:22]: is `jq` installed by default?

**Review by aramirez-es — APPROVED** [review]

File: files/healthcheck.sh (hunk 3 above)
Comment (fiunchinho) [reply files/healthcheck.sh:22]: Yes, since this version https://coreos.com/releases#976.0.0

File: troposphere/asg-elb-dns.py (hunk 2 above)
Comment (fiunchinho) [reply troposphere/asg-elb-dns.py:35]: I'll re-make the docs in another PR

**Pull request merged**